torch/csrc/jit/script/python_sugared_value.cpp

```diff
@@ -273,7 +269,7 @@ std::shared_ptr<SugaredValue> ModuleValue::attr(
 
   const auto& selfType = selfType_->expect<ClassType>();
 
-  if (selfType->hasAttribute(field) && selfType->getAttribute(field)->is_module()) {
+  if (const auto& t = selfType->findAttribute(field) && t->is_module()) {
```

Comment:
@suo is this what you meant?

Reply:
yeah sure

Reply:
Unfortunately this does not compile..I think we still have to keep this check.

Reply:
Okay let's drop it then. It's not particularly important.